LingoClash/LingoClash/Model/Games/PKGameMove.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  PKGameMove.swift
+//  LingoClash
+//
+//  Created by Sherwin Poh on 3/4/22.
+//
+
+import Foundation
+
+struct PKGameMove {
+    let id: Identifier
+    let question: Question
+    let player: Profile
+    let isCorrect: Bool
+    let timeTaken: Double
+    
+    // TODO: Possibly add a constant to represent a placeholder
+    init(question: Question, player: Profile, isCorrect: Bool, timeTaken: Double, id: Identifier = "") {
+        self.id = "Placeholder"
```

Comment:
Need advice on this: @CrownKira / however reviews this. I need to create a PKGameMove locally, what should the id be?

Reply:
Ok i will look into this. Perhaps we can make the id an optional field